test/include/kernels/MapMultiplyCoupledVars.h

```diff
@@ -26,6 +26,9 @@ class MapMultiplyCoupledVars : public ADKernel
   const ADVariableValue & _w;
   const std::map<std::string, Real> & _coupled_map;
   const std::map<std::string, std::string> & _dummy_string_to_string_map;
+  const std::map<dof_id_type, unsigned int> & _dummy_dofid_to_uint_map;
```

Comment:
```suggestion
  const std::map<dof_id_type, unsigned int> & _dummy_dofid_to_uint_map;
```
so this is unsigned int right now?
what are we trying to do here? Make sure MOOSE is always compiled with libmesh defines a high enough type for this?

Reply:
A good question: dof_id_type will depend on the build, it could be unsigned int, or unsigned long long. I can change it to unsigned long long. I used it because that is my use case. I will change it to unsigned long long.

Reply:
ok that's fine then. Let's just test it for a number that will always be supported, which is how it is currently.